packages/@angular/cli/tasks/serve.ts

```diff
@@ -182,7 +182,11 @@ export default Task.extend({
           return reject(err);
         }
         if (serveTaskOptions.open) {
-            opn(serverAddress);
+           if (serveTaskOptions.browser) {
+            opn(serverAddress, { app: serveTaskOptions.browser });
```

Comment:
This is platform specific.  Should it be normalized to allow consistent cross-platform behavior? Maybe make it consistent with protractor/webdriver?